services/api/service.go

```diff
@@ -449,6 +465,45 @@ func (api *RelayAPI) startValidatorRegistrationDBProcessor() {
 	}
 }
 
+func (api *RelayAPI) processPayloadAttributes(payloadAttributes beaconclient.PayloadAttributesData) {
+	// only for builder-api and if using see subscriptions instead of querying for payload attributes
+	if api.opts.BlockBuilderAPI && api.ffDisableQueryPayloadAttributes {
```

Comment:
rather than nest when the condition is met, better to exit early when it's not met